### ISSUE TITLE
German Translation for delsms cmd

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#88]
```
just a reminder for me: 
http://code.google.com/p/gtalksms/source/detail?r=74c33ba6aed69e3810a4cbbb7f13e5
15bc0682a1
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 8 Feb 2011 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
<string name="chat_help_del_sms_all">- %s: Lösche alle SMS ohne 
Nachfrage!</string>
<string name="chat_help_del_sms_sent">- %s: Lösche alle gesendeten SMS ohne 
Nachfrage!</string>
 <string name="chat_help_del_sms_contact">- %s: Lösche alle SMS zwischen mir und dem gewählen Kontakt ohne Nachfrage!</string>


<string name="chat_del_sms_syntax">Syntax 
is:\ndelsms:contact:[contact]\ndelsms:sent\ndelsms:all</string> (not sure here)
<string name="chat_del_sms_from">Lösche SMS von %s.</string>
<string name="chat_del_sms_nb">%d SMS gelöscht.</string>
<string name="chat_del_sms_error">SMS konnte nicht gelöscht werden.</string>
```

Original comment by `ThDietri...@gmail.com` on 8 Feb 2011 at 11:55

**Comment by GoogleCodeExporter** [comment]
```
ThDietrich86: thanks :)
```

Original comment by `fschm...@gmail.com` on 8 Feb 2011 at 2:14
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
http://code.google.com/p/gtalksms/source/detail?r=a389eeb47e2834fe088978a4006758
f11721a42c#
```

Original comment by `fschm...@gmail.com` on 8 Feb 2011 at 10:13
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 9 Feb 2011 at 8:05
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9.1
```

Original comment by `Florent....@gmail.com` on 13 Feb 2011 at 12:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
